src/Jackett.Common/Definitions/anilibria.yml

```diff
@@ -72,6 +73,8 @@ search:
       filters:
         - name: replace
           args: [" - EФильм", " - MOVIE"]
+        - name: replace
+          args: [" - EOVA", " - OVA"]
```

Comment:
I think I was trying to avoid the (granted, extremely unlikely) case of there being `EOVA` or `EФильм` as a word somewhere else in the title. Any benefit of using regex over a simple replace?